django/common/normalizers/normalizers.py

```diff
@@ -0,0 +1,18 @@
+from decimal import Decimal
+
+
+def normalize_to_bool(value: str):
+    if value.lower() in ("true", "1"):
+        return True
+    elif value.lower() in ("false", "0"):
+        return False
+    raise ValueError(f"cannot cast {value} to boolean")
+
+
+def normalize_to_str(value, accept_decimal=False):
```

Comment:
You forgot to set `accept_decimal` to True in `attribute.py`

Also, I'm not a huge fan a bool param that changes the behavior of a function but why not here 🤷‍♂️